src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -424,6 +424,113 @@ function ( WP_HTML_Token $token ): void {
 		};
 	}
 
+	/**
+	 * Creates a fragment processor at the current node.
+	 *
+	 * HTML Fragment parsing always happens with a context node. HTML Fragment Processors can be
+	 * instantiated with a `BODY` context node via `WP_HTML_Processor::create_fragment( $html )`.
+	 *
+	 * The context node may impact how a fragment of HTML is parsed. For example, consider the HTML
+	 * fragment `<td />Inside TD?</td>`.
+	 *
+	 * With a BODY context node results in the following tree:
```

Comment:
```suggestion
	 * With a BODY context node, this results in the following tree:
```

 maybe?

Reply:
It doesn't sound great, you're right.

```suggestion
	 * A BODY context node will produce the following tree:
```